pylint/checkers/exceptions.py

```diff
@@ -219,8 +219,6 @@ def visit_instance(self, instance: objects.ExceptionInstance) -> None:
     visit_exceptioninstance = visit_instance
 
     def visit_classdef(self, node: nodes.ClassDef) -> None:
```

Comment:
Something needs to be done here, unless ``instance._proxied `` always return a nodes.ClassDef. But I think at least ``Instance`` is a possibility.

Reply:
No. I fixed the typing here. `_proxied` is always `ClassDef`.